Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs

```diff
@@ -69,6 +77,7 @@ public void ApplySettingsForOpaqueDisplay()
             Debug.Log("Display is Opaque");
             Camera.main.clearFlags = CameraClearFlags_OpaqueDisplay;
             Camera.main.nearClipPlane = NearClipPlane_OpaqueDisplay;
+            Camera.main.backgroundColor = BackgroundColor_OpaqueDisplay;
```

Comment:
If solid color isn't set for the clear flags, does changing this value do anything that could be unwanted?

Reply:
no.